index.html

```diff
@@ -1,246 +1,260 @@
+
+                        <li class="li_class"> CSS</li>
+                        <li class="li_class"> Html</li>
+
+                    </ul>
+                    <div class="about_img about_img_1"></div>
                 </div>
-            </section>
+                <div class="about_row about_row_2">
+                    <ul class="skill_set">
+                        <li class="about_skills text_body_font"> Frameworks </li>
 
-            <section  class="about_section">
-                <div class="project_text text_color text_header_font">
-                    <h2>About</h2>
+                        <li class="li_class"> CSS</li>
+                        <li class="li_class"> Html</li>
+                    </ul>
+                    <div class="about_img about_img_2"></div>
                 </div>
-                <div class="text_subject text_color text_body_font about_text">
-                    <p>
-                        Hello!I am a software developer! I can 
-                        help you build a product, feature or website.
-                        Take a look of my works. If you like what you see 
-                        and have a project you need coded, don't hesitate 
-                        and contact me.
-                    </p>
+                <div class="about_row about_row_3">
+                    <ul class="skill_set">
+                        <li class="about_skills text_body_font"> Skills </li>
+                        <li class="li_class"> Github</li>
+                        <li class="li_class"> Terminal</li>
+                    </ul>
+                    <div class="about_img about_img_3"></div>
                 </div>
+            </div>
+
+        </section>
 
-                <a class="text_button text_body_font about_button">
-                    <p> Get my CV</p>
-                    <div class="download_image"></div>  
+        <!-- foooter -->
+        <footer class="contact_section">
+            <div class="mobile_logo"></div>
+
+            <h4 class="title-contact text_header_font">Get Started</h4>
+
+            <p class="section-desc text_body_font">
+                Hello!I am a software developer! I can
+                help you build a product, feature or website.
+                Take a look of my works. If you like what you see
+                and have a project you need coded, don't hesitate
+                and contact me.
+            </p>
+            <form action="https://formspree.io/f/mzbovllg" method="post">
+  <input
+    type="text"
+    name="FullName"
+    placeholder="Enter Your Name"
+    maxlength="30"
+    required/>
+
+    <input type="email" name="Email" placeholder="Enter Your Email" required />
+    <textarea
+        name="Message"
+        placeholder="Enter Your Message"
+        maxlength="500"
+        required>
+        </textarea>
+    <button class="" type="submit" name="submit-btn">Start collaboration</button>

```

Comment:
- [ ] Kindly your design overflows the page width, and there is scrolling, and that's because 
1-The fixed width for this button, kindly change it from `width: 390px;` to `width: 90%;` something with percentage. at `index.css line 101`.

![image](https://user-images.githubusercontent.com/35535130/160908086-fef56705-ee97-4e02-800e-17fc68727652.png)



2-This header makes your page overflow too, kindly make it smaller.

![image](https://user-images.githubusercontent.com/35535130/160908493-29393cf9-62a3-4d65-b0d7-9d589dfc0b0f.png)


Your page

![image](https://user-images.githubusercontent.com/35535130/160907865-288aeee4-1e45-4671-b411-2c91a740b846.png)

Reply:
@keroloussamy Thank you for the feedback